src/openforms/forms/tests/e2e_tests/test_translation_warnings.py

```diff
@@ -0,0 +1,479 @@
+
+            warning_list = page.locator("css=.messagelist")
+            await warning_list.get_by_role("link", name="4 translation").click()
+
+            modal = page.locator("css=.react-modal__content")
+
+            await expect(
+                modal.get_by_role("row", name="Literals	English	begin text")
+            ).to_be_visible()
+            await expect(
+                modal.get_by_role("row", name="Literals	English	step previous text")
+            ).to_be_visible()
+            await expect(
+                modal.get_by_role("row", name="Literals	English	change text")
+            ).to_be_visible()
+            await expect(
+                modal.get_by_role("row", name="Literals	English	confirm text")
+            ).to_be_visible()
+
+    async def test_no_warning_with_only_english_button_translations(self):
+        @sync_to_async
+        def setUpTestData():
+            form = FormFactory.create(
+                name="Playwright test",
+                name_nl="Playwright test nl",
+                name_en="Playwright test en",
+                explanation_template_nl="playwrite test nl",
+                explanation_template_en="playwrite test en",
+                begin_text_nl="",
+                begin_text_en="playwrite test en",
+                previous_text_nl="",
+                previous_text_en="playwrite test en",
+                change_text_nl="",
+                change_text_en="playwrite test en",
+                confirm_text_nl="",
+                confirm_text_en="playwrite test en",
+                generate_minimal_setup=True,
+                formstep__form_definition__name_nl="Playwright test nl",
+                formstep__form_definition__name_en="Playwright test en",
+                formstep__form_definition__configuration={
+                    "components": [
+                        {
+                            "type": "textfield",
+                            "key": "someField",
+                            "label": "Some Field",
+                        }
+                    ],
+                },
+                formstep__form_definition__component_translations={
+                    "en": {
+                        "Some Field": "Some EN Field",
+                    },
+                    "nl": {
+                        "Some Field": "Een NL Veld",
+                    },
+                },
+                translation_enabled=True,
+            )
+            return form
+
+        await create_superuser()
+        form = await setUpTestData()
+        admin_url = str(
+            furl(self.live_server_url)
+            / reverse("admin:forms_form_change", args=(form.pk,))
+        )
+
+        async with browser_page() as page:
+            await self._admin_login(page)
+            await page.goto(str(admin_url))
+
+            warning_list = page.locator("css=.messagelist")
+
+            await expect(warning_list).not_to_be_visible()
```

Comment:
Yes exactly because those translations have fallback fields it should trigger this bit of code
![Screenshot from 2023-08-31 12-59-36](https://github.com/open-formulieren/open-forms/assets/101265650/532f83fa-ee75-4b59-9ae4-875500be3a29)